weblate/trans/views/basic.py

```diff
@@ -512,6 +512,13 @@ def server_error(request):
         return django.views.defaults.server_error(request)
 
 
+def csrf_error(request, **kwargs):
+    """
+    Error handler for csrf errors.
+    """
+    return denied(request)
```

Comment:
This function seems to be unused, you're using directly denied view in CSRF_FAILURE_VIEW...